<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Configuration</string>
     </attribute>
     <widget class="QLineEdit" name="ipName">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="portName">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxUDP">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>86</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>UDP</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxTCP">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>86</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>TCP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>55</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PORT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setCfg">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>240</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Configuration</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SenseHatData">
     <attribute name="title">
      <string>Sense Hat Data</string>
     </attribute>
     <widget class="QPushButton" name="getTemp">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Get</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="tempNum">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>55</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Résulat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="getPre">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Get</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Pressure</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>55</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Résulat</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="preNum">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>8x8 Led Data Matrix</string>
     </attribute>
     <widget class="QLineEdit" name="textToDisplay">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="displayText">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Display Text</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
